test/first_party/CMakeLists.txt

```diff
@@ -66,6 +66,9 @@ if (NOT CROSS_COMPILER_COMMAND_RV32 OR NOT CROSS_COMPILER_COMMAND_RV64)
     message(FATAL_ERROR "No suitable cross-compiler found. We recommend downloading Clang from https://github.com/llvm/llvm-project/releases")
 endif()
 
+message(STATUS "Compiling RV32 tests with: ${CROSS_COMPILER_COMMAND_RV32}")
+message(STATUS "Compiling RV64 tests with: ${CROSS_COMPILER_COMMAND_RV64}")
```

Comment:
I think `$<JOIN:${CROSS_COMPILER_COMMAND_RV32}, >` Should work, but probably not worth fixing.

Reply:
Ah didn't know about that. I'll do it in a future PR if I remember...